csvqb/csvqb/tests/unit/configloaders/infojsontests.py

```diff
@@ -21,33 +23,160 @@ def test_csv_cols_assumed_dimensions(self):
         data = pd.read_csv(self.get_test_cases_dir() / "configloaders" / "data.csv")
         cube = get_cube_from_info_json(self.get_test_cases_dir() / "configloaders" / "info.json", data)
 
-        matching_columns = [c for c in cube.columns if c.csv_column_title == "Undefined Column"]
-        self.assertEqual(1, len(matching_columns))
-        undefined_column_assumed_definition: CsvColumn = matching_columns[0]
+        # matching_columns = [c for c in cube.columns if c.csv_column_title == "Undefined Column"]
```

Comment:
Let's make sure to uncomment this again!

Reply:
uncommented all the commented lines of code

Reply:
Great, make sure you commit & push the change.